docs/configure-containerd-shim-runsc-v1.md

```diff
@@ -1,13 +1,17 @@
 # Configure containerd-shim-runsc-v1 (Shim V2)
 
-This document describes how to configure runtime options for `containerd-shim-runsc-v1`.
-This is follows on to the instructions of [Runtime Handler Quick Start (shim v2) (containerd >=1.2)](runtime-handler-shim-v2-quickstart.md) and requires containerd 1.3 or later.
+This document describes how to configure runtime options for
+`containerd-shim-runsc-v1`.  This is follows on to the instructions of \[Runtime
+Handler Quick Start (shim v2) (containerd
 
-## Configuration
+> =1.2)](runtime-handler-shim-v2-quickstart.md) and requires containerd 1.3 or
+> later.
```

Comment:
Is the formatting on a link here broken?

Reply:
Done.